client/app/common/less/common.less

```diff
@@ -1,5 +1,9 @@
 @text-disabled-color: #757575;
 
+.ovh-font {
+    font-family: "ovh-font";
+}
+
```

Comment:
It's added because of Bootstrap's `font-family: inherit;` that overrides every `font-family` injected by ovh-font

Reply:
I think it's not the good way to resolve this. For example : 
https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/telecom/pack/slots/informations/pack-informations.html#L18
It's the `h5` class that adds the wrong behaviour. If the `div.oui-tile__description` was replaced by an `<h5>` tag and the `h5` class is removed, the result would be correct.

Reply:
Ok, so all icons that contains `h5`or `h6` class are now tags.